src/models/krate.rs

```diff
@@ -539,36 +517,37 @@ impl Crate {
     }
 }
 
-pub struct OwnerInviteEmail {
-    /// The destination email address for this email.
-    recipient_email_address: String,
+/// Details of a newly created invite.
+#[derive(Debug)]
+pub enum NewOwnerInvite {
+    /// The invitee was a [`User`], and they must accept the invite through the
+    /// UI or via the provided invite token.
+    User(User, Secret<String>),
 
-    /// Email body variables.
-    user_name: String,
-    domain: String,
-    crate_name: String,
-    token: SecretString,
+    /// The invitee was a [`Team`], and they were immediately added as an owner.
+    Team(Team),
 }
```

Comment:
The model now returns a success enum that describes what type of owner was added, and if it is a user, it includes the acceptance token.